DEVELOPMENT.md

```diff
@@ -130,19 +129,42 @@ By default, KubeArchive listens to `Event`s in the `test` namespace.
     ```
 
 1. **[ Optional ]** Create a service account with a specific role to test the REST API.
-   This Helm chart already provides `kubearchive-test-sa` with `view` privileges for testing purposes.
+    This Helm chart already provides `kubearchive-test-sa` with `view` privileges for testing purposes.
 
 1. On a new terminal, use `curl` or your browser to perform a query:
     ```bash
     curl -s --cacert ca.crt -H "Authorization: Bearer $(kubectl create token kubearchive-test -n kubearchive)" \
-   https://localhost:8081/apis/batch/v1/jobs | jq
+    https://localhost:8081/apis/batch/v1/jobs | jq
     ```
 
 1. Check the new logs on the KubeArchive API:
     ```bash
     kubectl logs -n kubearchive -l app=kubearchive-api-server
     ```
 
+## Use the KubeArchive CLI
+
+1. Use `kubectl` to port forward, this will keep the terminal occupied:
+    ```bash
+    kubectl port-forward -n kubearchive svc/kubearchive-api-server 8081:8081
+    ```
+
+1. Get the Certificate Authority (CA) from the `kubearchive-api-server-tls` secret:
+    ```bash
+    kubectl get -n kubearchive secrets kubearchive-api-server-tls -o jsonpath='{.data.ca\.crt}' | base64 -d > ca.crt
+    ```
+
+1. Run the CLI:
+    ```bash
+    go run cmd/kubectl-archive/main.go get batch/v1/jobs --token $(kubectl -n kubearchive create token kubearchive-test)
```

Comment:
I was reading the whole DEVELOPMENT.md, and we mentioned the namespaces: `test` and kubearchive-test`. In the charts, we have defined  the `test` [namespace](https://github.com/kubearchive/kubearchive/blob/a00ecced3f7c18911558f55cdd25a76ee9b91e57/charts/kubearchive/templates/test/namespace.yaml). 
Maybe we need to add the creation of that namespace before these steps, or should we change it to `test` ?

Reply:
`kubearchive-test` is the service account name, the namespace is `kubearchive`

we could move `kubearchive-test` to the `test` namespace though

Reply:
My bad, when I read this, I read: `-n kubearchive-test` :/